config/initializers/014-track-setting-changes.rb

```diff
@@ -29,6 +29,21 @@
     end
   end
 
+  # Set bootstrap min users for private sites to a lower default
+  PRIVATE_BOOTSTRAP_MODE_MIN_USERS ||= 10.freeze
```

Comment:
Why `||=` and `freeze`? Can those be dropped and the definition moved to the top of the file?

Reply:
Probably don't need the freeze, I believe I was getting this CONSTANT was already initiated warning or something so I added the `||=`.